packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.9.19+1
 
+* Corrects pigeon configuration output paths
```

Comment:
This and the version change should be reverted; this only affects development of the package, and so falls under one of our documented exemptions for version and CHANGELOG changes.